python/datasources/cdc_restricted.py

```diff
@@ -19,39 +42,231 @@ def upload_to_gcs(self, _, **attrs):
             'upload_to_gcs should not be called for CDCRestrictedData')
 
     def write_to_bq(self, dataset, gcs_bucket, **attrs):
-        gcs_files = self.get_attr(attrs, 'filename')
-
-        # In this instance, we expect filename to be a string with
-        # comma-separated CSV filenames.
-        if ',' not in gcs_files:
-            raise ValueError('filename passed to write_to_bq is not a '
-                             'comma-separated list of files')
-        files = gcs_files.split(',')
-        print("Files that will be written to BQ:", files)
-
-        # For each of the files, we load it as a dataframe and add it as a
-        # table in the BigQuery dataset. We expect that all aggregation and
-        # standardization of the data has been done by this point.
-        int_cols = [std_col.COVID_CASES, std_col.COVID_HOSP_Y,
-                    std_col.COVID_HOSP_N, std_col.COVID_HOSP_UNKNOWN,
-                    std_col.COVID_DEATH_Y, std_col.COVID_DEATH_N,
-                    std_col.COVID_DEATH_UNKNOWN]
-        for f in files:
-            # Explicitly specify county_fips is a string.
+        for geo in ['state', 'county']:
+            for demo in ['sex', 'race', 'age']:
+                filename = f'cdc_restricted_by_{demo}_{geo}.csv'
+                df = gcs_to_bq_util.load_csv_as_df(
+                    gcs_bucket, filename, dtype={'county_fips': str})
+
+                df = self.generate_breakdown(df, demo, geo)
+
+                if demo == 'race':
+                    std_col.add_race_columns_from_category_id(df)
+
+                column_types = get_col_types(df)
+
+                table_name = f'by_{demo}_{geo}_processed'
+                gcs_to_bq_util.add_df_to_bq(
+                    df, dataset, table_name, column_types=column_types)
+
+        for filename in EXTRA_FILES:
             df = gcs_to_bq_util.load_csv_as_df(
-                gcs_bucket, f, dtype={'county_fips': str})
+                gcs_bucket, filename, dtype={'county_fips': str})
+
+            self.clean_frame_column_names(df)
+
+            int_cols = [std_col.COVID_CASES, std_col.COVID_HOSP_Y,
+                        std_col.COVID_HOSP_N, std_col.COVID_HOSP_UNKNOWN,
+                        std_col.COVID_DEATH_Y, std_col.COVID_DEATH_N,
+                        std_col.COVID_DEATH_UNKNOWN]
 
-            # All columns are str, except outcome columns.
             column_types = {c: 'STRING' for c in df.columns}
             for col in int_cols:
                 if col in column_types:
                     column_types[col] = 'FLOAT'
+
             if std_col.RACE_INCLUDES_HISPANIC_COL in df.columns:
                 column_types[std_col.RACE_INCLUDES_HISPANIC_COL] = 'BOOL'
 
-            # Clean up column names.
-            self.clean_frame_column_names(df)
+            print('uploading extra file')
```

Comment:
are we leaving these print statements in as feedback when the script is run?

Reply:
Yeah i just thought it was nice to see what was going on. We should probably add a real logging library at some point.